infra/pr_helper.py

```diff
@@ -223,6 +228,26 @@ def get_pull_request_number(self, commit):
       return None
     return pr_response.json()[0]['number']
 
+  def get_past_contributors(self, project_path):
+    """Returns a list of past contributors of a certain project."""
+    commits_response = requests.get(f'{BASE_URL}/commits?path={project_path}',
+                                    headers=self._headers)
+
+    if commits_response.ok:
+      commits = commits_response.json()
+      contributors = set()
+      verified_contributors = []
+      for commit in commits:
```

Comment:
Should we have a limit on the number of commits we process, in case this grows too large for us to handle quickly?